hack/verify-api-reference-docs.sh

```diff
@@ -26,11 +26,19 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 API_REFERENCE_DOCS_ROOT="${KUBE_ROOT}/docs/api-reference"
+# Use REPO_DIR if provided so we can set it to the host-resolvable path
+# to the repo root if we are running this script from a container with
+# docker mounted in as a volume.
+# We pass the host output dir to update-api-reference-docs.sh, but use
```

Comment:
Correct. They point to the same files, you just need a different absolute path to resolve them on the host vs from inside the test container.